Description: Use pkg-config for libpcsclite linkage flags
 At least in debian, we can rely on pkg-config being available and
 returning more accurate ldflags.
Author: Reinhard Tartler <siretart@tauware.de>
---
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -776,7 +776,7 @@ ifdef CONFIG_NATIVE_WINDOWS
 #dynamic symbol loading that is now used in pcsc_funcs.c
 #LIBS += -lwinscard
 else
-LIBS += -lpcsclite -lpthread
+LIBS += $(shell pkg-config --libs libpcsclite)
 endif
 endif
 
